news/deprecate_py2.rst

```diff
@@ -0,0 +1,3 @@
+**Deprecated:**
```

Comment:
Should this be removed instead of deprecated?

Reply:
Changed